code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -267,3 +267,11 @@
 		..()
 		reagents.add_reagent("vinegar", 50)
 
+/obj/item/weapon/reagent_containers/food/condiment/exotic
+	name = "Exotic Bottle"

```

Comment:
I wasn't arguing! You don't have to be defensive. I just don't have time to work on more code for a few more hours.